app/config/FrontendAppConfig.scala

```diff
@@ -129,8 +129,9 @@ class FrontendAppConfig @Inject()(configuration: Configuration, servicesConfig:
       )
   }"
 
-  lazy val validEvent1Header: String = configuration.get[String]("validEvent1Header")
-  lazy val validEvent6Header: String = configuration.get[String]("validEvent6Header")
-  lazy val validEvent22Header: String = configuration.get[String]("validEvent22Header")
-  lazy val validEvent23Header: String = configuration.get[String]("validEvent23Header")
+  val validEvent1Header: String = configuration.get[String]("validEvent1Header")
```

Comment:
Why did we stop using `lazy val`?

Reply:
It was after some bug hunting that Pavel and I did - `lazy` was stopping a useful error message being shown to say that the value couldn't be found in the config. Glad you spotted that though because I should change the other values too 👍